front-end/src/components/main-page/TranscriptDescription.jsx

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+import React, { Component } from 'react';
+import {exampleTranscript} from "../helpers/ExampleTranscript.js"; 
+
+class TranscriptDescription extends Component {
+	render() {
+		return (
+            <div className='whitespace-nowrap'>
+                <p className='text-center font-bold'> 
+                    Required Data Format:
+                </p>
+                <pre className='overflow-auto leading-8 text-sm'>
```

Comment:
I think we can add one tooltip explaining all parts. I don't think it's possible to do that line by line at least using <pre> tag and json because it will display the json regardless of the content

Reply:
I believe we would be able to add <span> tags around each line which would then work with a tooltip. If we cannot, then I'd be fine with 1 large tooltip with a quick summary.